cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/TransactionHttpHandler.java

```diff
@@ -323,7 +323,10 @@ private boolean initializePruningDebug(HttpResponder responder) {
       } catch (ClassNotFoundException | InstantiationException | IllegalAccessException |
         ClassCastException | IOException e) {
         LOG.debug("Not able to instantiate pruning debug class", e);
```

Comment:
should this be error()?
Also, it may be safer to this on unchecked exceptions, too.

Reply:
I left the log as debug, since it is an HTTP handler. We can always enable debug logging on the class using REST APIs to see the logs.
The unchecked exceptions will propagate over to the HttpExceptionHandler.